### ISSUE TITLE
Looking for someone with the privileges to create the [depclean] tag on StackOverflow.com

### ISSUE THREAD

**Issue by cesarsotovalero** [castor-software/depclean#51]
In StackOverflow, please create the tag **[depclean]** to post any question on the usage of DepClean.
The tag is not available at the moment, and you need at least 1500 reputation points to create the tag.
Once the tag is created, all the questions should appear in [https://stackoverflow.com/questions/tagged/depclean](https://stackoverflow.com/questions/tagged/depclean)

**Comment by ABHAY0O7** [comment]
Hello @cesarsotovalero,
I recently got some problem in using depclean-maven-plugin.
So, I decided to post it on StackOverflow but I am unable to use the tag `depclean`.
It is showing this
`Creating the new tag 'depclean' requires at least 1500 reputation. Try something from the existing tags list instead.`
Do I have to earn 1500 reputations to use the tag depclean?
Also when I tried to search `depclean` in the tags then there is no existing tag named `depclean`.
Please help me with this.

Thank you.

**Comment by ABHAY0O7** [comment]
Hey! @cesarsotovalero
I ask a question regarding depclean [here](https://stackoverflow.com/questions/67010450/dependencies-where-not-copied-locally-depclean-warning) 
But I was not able to use the tag `depclean`.
I hope you will help me out with this question as soon as possible.

Thank you
Abhay

**Comment by cesarsotovalero** [comment]
> Hello @cesarsotovalero,
> I recently got some problem in using depclean-maven-plugin.
> So, I decided to post it on StackOverflow but I am unable to use the tag `depclean`.
> It is showing this
> `Creating the new tag 'depclean' requires at least 1500 reputation. Try something from the existing tags list instead.`
> Do I have to earn 1500 reputations to use the tag depclean?
> Also when I tried to search `depclean` in the tags then there is no existing tag named `depclean`.
> Please help me with this.
> 
> Thank you.

Hi @ABHAY0O7, thanks for pointing this out. You're definitely right, the `depclean` tag doesn't exist in SO, and to create it [one needs at least 1500 points of reputation](https://stackoverflow.com/help/privileges/create-tags). I have only 98 points right now, so I cannot create the tag at the moment. I'll edit this issue and let it open with the hope that someone with the necessary points creates the tag for us (or I got the 1500 points 😄 ).

**Comment by cesarsotovalero** [comment]
> Hey! @cesarsotovalero
> I ask a question regarding depclean [here](https://stackoverflow.com/questions/67010450/dependencies-where-not-copied-locally-depclean-warning)
> But I was not able to use the tag `depclean`.
> I hope you will help me out with this question as soon as possible.
> 
> Thank you
> Abhay

Hi @ABHAY0O7, thanks for asking this question in SO. 
I'll answer your question there 👍

**Comment by ABHAY0O7** [comment]
Hi @cesarsotovalero, 
I want to clarify that the problem I asked about in stackoverflow got solved.
The problem is occurring because I run the project on my C-drive (default) where probably depClean doesn't have any write access to it. Therefore, I tried to run the project on my D-drive, and boom! it runs successfully without any warning.
I think this information may be added to the README.

Thank you
